cmd/notation/cert.go

```diff
@@ -341,17 +345,26 @@ func showCerts(opts *certShowOpts) error {
 	if len(certs) == 0 {
 		return fmt.Errorf("%s not found", path)
 	}
-	showRootCA(certs[len(certs)-1])
+
+	//write out
+	fmt.Println("Display certificate details. Starting from leaf certificate if it's a certificate chain.")
+	fmt.Println("-----------------------------------------------------------------------------------------")
+	for ind, cert := range certs {
+		showCert(cert)
+		if ind != len(certs)-1 {
+			fmt.Println("-----------------------------------------------------------------------------------------")
+		}
+	}
```

Comment:
Do you think we need to wrap these to a function like `DisplayCerts`?

Reply:
Oh yeah, sure. will update.